app/src/main/java/com/google/jetpackcamera/MainActivity.kt

```diff
@@ -67,6 +68,8 @@ import kotlinx.coroutines.flow.onEach
 import kotlinx.coroutines.launch
 
 private const val TAG = "MainActivity"
+const val FIRST_FRAME_TRACE = "FirstFrameTrace"
+const val FIRST_FRAME_COOKIE = 12345
```

Comment:
Assuming we only start 1 first frame trace, we don't really need a cookie. You can just put 0 for the cookie in `begin/endAsyncSection`.